src/main/java/uk/gov/hmcts/reform/pip/account/management/service/PublicationService.java

```diff
@@ -37,19 +38,41 @@ public class PublicationService {
      * @param surname - surname
      * @return string for logging success or failure
      */
-    public String sendNotificationEmail(String emailAddress, String forename, String surname) {
+    public boolean sendNotificationEmail(String emailAddress, String forename, String surname) {
         JSONObject jsonObject = new JSONObject();
         jsonObject.put("email", emailAddress);
         jsonObject.put("forename", forename);
         jsonObject.put("surname", surname);
         try {
-            return webClient.post().uri(new URI(url + "/notify/created/admin"))
+            log.info(webClient.post().uri(url + "/notify/created/admin")
                 .body(BodyInserters.fromValue(jsonObject)).retrieve()
-                .bodyToMono(String.class).block();
-
-        } catch (WebClientException | URISyntaxException ex) {
+                .bodyToMono(String.class).block());
+            return true;
+        } catch (WebClientException ex) {
             log.error(String.format("Request failed with error message: %s", ex.getMessage()));
-            return "Email request failed to send: " + emailAddress;
+            return false;
+        }
+    }
+
+    /**
+     * Method calling Publication services send welcome email api.
+     * @param emailAddress email address to send welcome email to
+     * @param isExisting bool to determine if coming from migration or new user creation
+     * @return success message for logging
+     */
+    public boolean sendMediaNotificationEmail(String emailAddress, boolean isExisting) {
+        JSONObject body = new JSONObject();
+        body.put("email", emailAddress);
+        body.put("isExisting", isExisting);
+        try {
+            log.info(webClient.post().uri(url + WELCOME_EMAIL_URL)
+                .bodyValue(body)
```

Comment:
This will need the client registration ID for publication services :)

Reply:
does it not have that as the default from the webclient config?